include/swift/AST/ASTContext.h

```diff
@@ -904,10 +911,8 @@ class ASTContext final {
 
   /// Check whether the module with a given name can be imported without
   /// importing it.
-  ///
-  /// Note that even if this check succeeds, errors may still occur if the
-  /// module is loaded in full.
```

Comment:
Was it intentional to remove this comment? Does it no longer apply to this function, only to `canImportModuleImpl()`?